blscosi/protocol/struct.go

```diff
@@ -23,6 +23,12 @@ const DefaultProtocolName = "blsCoSiProtoDefault"
 // main protocol.
 const DefaultSubProtocolName = "blsSubCoSiProtoDefault"
 
+// AddPrefix adds a prefix to msg in order to prevent the use of
+// BLSCoSi service as an oracle.
+func AddPrefix(msg *[]byte) {
```

Comment:
Use lowercase i.e. `addPrefix` because there's no value in exporting this for use outside of the package.

In general, we default to private because that preserves our ability to evolve the internals of packages without disrupting callers.

Reply:
done in a6b6462f5b7b3738cd3402ffeea9922a07af7a25